### PR TITLE
Introduce the Example page

### PR DESCRIPTION
**Pull request by VasilyStrelyaev**
\cc @DevExpress/testcafe-docs 
Almost as the designer painted

image

fixes DevExpress/testcafe#506

### CODE REVIEW

**Comment by VasilyStrelyaev** [comment]
Hello-o-o!
Can you he-e-ear me?..

**Comment by inikulin** [comment]
lgtm

**Comment by churkin** [comment]
lgtm

**Comment by AlexanderMoskovkin** [comment]
lgtm

**Comment by MargaritaLoseva** [comment]
lgtm

File: src/css/example-page.scss [hunk 1]
```diff
@@ -0,0 +1,184 @@
...
+    padding-bottom: 10px;
+}
+
+label[for="tried-test-cafe"] {
+    font-weight: bold;
+}
+
+.main-content {
+    width: 850px;
+    margin-right: auto;
```
Comment (kirovboris) [inline src/css/example-page.scss]: Please use single css rule for definition ```padding``` and ```margin```.
```css
margin: 0 0 30px auto;
padding: 39px 30px 0 30px;
```
etc.

**Comment by VasilyStrelyaev** [comment]
FPR
@kirovboris Could you please ensure I did it right everywhere? Visually, nothing is broken, but still...
I need time to get used to these shorthands...

**Comment by kirovboris** [comment]
lgtm

**Pull request merged**